plugins/CustomVariables/API.php

```diff
@@ -34,7 +34,6 @@ class API extends \Piwik\Plugin\API
     protected function getDataTable($idSite, $period, $date, $segment, $expanded, $flat, $idSubtable)
     {
         $dataTable = Archive::createDataTableFromArchive(Archiver::CUSTOM_VARIABLE_RECORD_NAME, $idSite, $period, $date, $segment, $expanded, $flat, $idSubtable);
-        $dataTable->filter('Sort', array(Metrics::INDEX_NB_ACTIONS, 'desc', $naturalSort = false, $expanded));
```

Comment:
No longer needed since we do now always sort in GenericFilter. So we do no longer sort the entire table twice